packages/client/lib/rpc/modules/eth.ts

```diff
@@ -931,7 +931,8 @@ export class Eth {
 
     // Add the tx to own tx pool
     const { txPool } = this.service.synchronizer as FullSynchronizer
-    txPool.add(tx)
+    // TODO handle any errors here
+    await txPool.add(tx)
```

Comment:
yep i think this just can be

```suggestion
    try {
      await txPool.add(tx)
    } catch (error: any) {
      throw {
        code: INVALID_PARAMS
        message: error?.message ?? error.toString()
      }
    }
```